obspy/core/event/base.py

```diff
@@ -1166,16 +1166,9 @@ class DataUsed(__DataUsed):
     moment-tensor inversion.
 
     :type wave_type: str or :class:`~obspy.core.event.header.DataUsedWaveType`
```

Comment:
Agree on this last one. I'll do that together with a squash/rebase to fix the commit history.